lib/DB/Schema/Result/ProblemSet/ReviewSet.pm

```diff
@@ -43,7 +43,7 @@ subroutine that returns the hashref for the valid parameters:  (currently empty)
 =cut
 
 sub valid_params ($=) {
-	return { test_param => qr/[01]/ };
+	return { test_param => 'bool' };
```

Comment:
What is a review set `test_param`?  Is this just something you added for testing purposes?  This and all of the other related code should be deleted.  Implement actual code that will be used for webwork 3.

Reply:
This was just a dummy param for testing.  I'll remove it.

Reply:
It was a pain to remove it, so I renamed it to `can_retake` thinking that we can allow users to retake a review set.

Reply:
For now that is okay.  I don't think that is a valid option either though.  If can_retake is false, then it should be a quiz.  Not a review set.

Reply:
We can talk about what defines a review set, but the idea was that it doesn't count for a grade and perhaps solutions are always available.

Reply:
Agreed and fixed.